src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc

```diff
@@ -46,9 +49,48 @@ aws:
                 # true/false - ask AWS to automatically decrypt SecureString type with KMS
                 useSecureParameters: true
                 # String - you can use a custom root path the application will look for configurations in. The default is '/config/application'.
-                rootHierarchyPath: /config/application
+                rootHierarchyPath: /config
 ----
 
 
+.Configuration Properties Example - application.yml
+[source,yaml]
+----
+micronaut:
+  application:
+    name: demo
+---
+endpoints:
+    beans:
+        enabled: true
+        sensitive: false
+    refresh:
+        enabled: true
+        sensitive: false
+----
+
+
+We can use the name of the application in the parameter store like so:
+
+AWS Parameter Store Key: /config/demo/simple
+AWS Parameter Store Value: some demo app specific value
+
+If you don't have an application name set, or just want a global config use a key name like this:
+
+/config/application/simple
+
+Or do environments only values:
+
+AWS Parameter Store Key: /config/application_dev/simple
+AWS Parameter Store Value: some dev only text
+
+AWS Parameter Store Key: /config/application_test/simple
+AWS Parameter Store Value: some test only text
+
+
+TIP: Keep in mind certain characters are not allowed for key names: https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-parameter-name-constraints.html
+
+Check the https://github.com/micronaut-projects/micronaut-examples/aws repository for a sample application.
 
+TIP: If you are using Lambda functions with Parameter Store, you will need to manually add the 'ec2' environment to the current environments.
```

Comment:
That is pending https://github.com/micronaut-projects/micronaut-core/pull/1469

Reply:
ok so should I reword this that 1.1.0 this is the case but 1.1.1+ it works? or just not mention it at all?

Reply:
This should be specified that it is only required for Micronaut 1.1.x or before